cmd/cli_test.go

```diff
@@ -80,40 +80,29 @@ func TestCli(t *testing.T) {
 			t.Errorf("There should be an error log, instead got: '%s'", outStr)
 		}
 
-		out, err = ioutil.ReadFile(outputFileName)
+		out, err = os.ReadFile(outputFileName)
 		if err != nil {
 			t.Fatal(err)
 		}
 
 		expected := `<index>
- <timestamp></timestamp>
+ <vendor>testing_vendor_index</vendor>
+ <url>test-continue-despite-errors.xml</url>
+ <>
  <pindex>
   <pdsc vendor="TheVendor" url="test/" name="ThePack2" version="1.1.0" timestamp=""></pdsc>
```

Comment:
I think timestamp is  optional and should not be printed and then left empty.
The valid timestamp should come from the vendor.pidx that this pdsc element was read from.